docs/markdown/snippets/json_output_format.md

```diff
@@ -0,0 +1,5 @@
+## Added 'json' output_format to configure_file()
+
+When no input file is specified, [[configure_file()]] can now
```

Comment:
My suggestion removed the `()` when using the `[[]]` link formatter. I'm not sure it matters for the output generation but we usually don't include that...